symfony/client/src/orangehrmTimePlugin/pages/project/Project.vue

```diff
@@ -202,21 +198,21 @@ export default {
       headers: [
         {
           name: 'customer',
-          title: 'Customer',
+          title: 'Customer Name',
           sortField: 'customer.name',
-          style: {flex: 2},
+          style: {flex: '15%'},
         },
         {
           name: 'project',
           slot: 'title',
           title: 'Project',
           sortField: 'project.name',
-          style: {flex: 3},
+          style: {flex: '15%'},
         },
         {
           name: 'projectAdmins',
           title: 'Project Admins',
-          style: {flex: 3},
+          style: {flex: '20%'},
```

Comment:
any special reason for percentage

Reply:
yes flex basis to have more space for project admins, since there can be many